flink-filesystems/flink-gs-fs-hadoop/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,86 @@
+- com.google.android:annotations:4.1.1.4
+- com.google.cloud.bigdataoss:gcs-connector:hadoop3-2.2.4
+- com.google.cloud.bigdataoss:gcsio:2.2.4
+- com.google.cloud.bigdataoss:util-hadoop:hadoop3-2.2.4
+- com.google.cloud.bigdataoss:util:2.2.4
+- com.google.cloud:google-cloud-core-http:2.3.3
+- com.google.cloud:google-cloud-core:2.3.3
+- com.google.cloud:google-cloud-storage:2.2.2
+- com.google.code.gson:gson:2.8.9
+- com.google.flogger:flogger-system-backend:0.7.1
+- com.google.flogger:flogger:0.7.1
+- com.google.flogger:google-extensions:0.7.1
+- com.google.guava:guava:31.0.1-jre
+- com.google.guava:failureaccess:1.0.1
+- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
+- com.google.http-client:google-http-client-apache-v2:1.40.1
+- com.google.http-client:google-http-client-appengine:1.40.1
+- com.google.http-client:google-http-client-gson:1.40.1
+- com.google.http-client:google-http-client-jackson2:1.40.1
+- com.google.http-client:google-http-client:1.40.1
+- com.google.j2objc:j2objc-annotations:1.3
+- com.google.oauth-client:google-oauth-client:1.32.1
+- com.google.errorprone:error_prone_annotations:2.9.0
+- commons-codec:commons-codec:1.15
+- io.grpc:grpc-alts:1.41.1
+- io.grpc:grpc-api:1.41.1
+- io.grpc:grpc-auth:1.41.1
+- io.grpc:grpc-core:1.41.1
+- io.grpc:grpc-context:1.42.1
+- io.grpc:grpc-grpclb:1.41.1
+- io.grpc:grpc-netty-shaded:1.41.1
+- io.grpc:grpc-protobuf-lite:1.41.1
+- io.grpc:grpc-protobuf:1.41.1
+- io.grpc:grpc-stub:1.41.1
+- io.opencensus:opencensus-api:0.28.0
+- io.opencensus:opencensus-contrib-http-util:0.28.0
+- io.perfmark:perfmark-api:0.23.0
+- org.apache.httpcomponents:httpclient:4.5.13
+- org.apache.httpcomponents:httpcore:4.4.14
+- org.conscrypt:conscrypt-openjdk-uber:2.5.1
+
+This project bundles the following dependencies under BSD-2 License (https://opensource.org/licenses/BSD-2-Clause).
+See bundled license files for details.
+
+- com.google.api:api-common:2.1.1
+- com.google.api:gax-httpjson:0.92.1
+- com.google.api:gax:2.7.1
+
+This project bundles the following dependencies under BSD-3 License (https://opensource.org/licenses/BSD-3-Clause).
+See bundled license files for details.
+
+- com.google.auth:google-auth-library-credentials:1.3.0
+- com.google.auth:google-auth-library-oauth2-http:1.3.0
+- com.google.protobuf:protobuf-java-util:3.19.1
+- com.google.protobuf:protobuf-java:3.19.1
+- org.threeten:threetenbp:1.5.2
+
+This project bundles the following dependencies under the GPL2 License (https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) and MIT License (https://opensource.org/licenses/MIT).
+See bundled license files for details.
+
+- org.checkerframework:checker-compat-qual:2.5.3
+- org.checkerframework:checker-qual:3.19.0
+
+This project bundles the following dependencies under the MIT License (https://opensource.org/licenses/MIT).
+See bundled license files for details.
+
+- org.codehaus.mojo:animal-sniffer-annotations:1.19
```

Comment:
please double-check whether these need to be bundled. They are afaik not used at runtime.